util/standardize_metadata.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+"""This script sorts the ordering of the metadata in each ontology markdown file.
+
+Run with: ``python standardize_metadata.py``.
```

Comment:
Can we add an action that runs this script on any PR if a yaml file is changed, and automatically pushes the new order?

Reply:
That's possible for branches off the origin. Not sure if you can arbitrarily have a bot pushing to forks. However, it's possible for maintainers to do it so I give it 75% likely to be okay

Reply:
Not sure how the best way to go about testing this is, though. For now, you can run `tox -e lint` to run the linter for the yaml locally.